trade-finance-manager-ui/server/controllers/helpers/amendments.helper.js

```diff
@@ -33,9 +33,52 @@ const userCanEditBankDecision = (amendment, user) => {
   return isPim && hasSubmittedDecision ? true : false;
 };
 
+/**
+ * @param {Object} amendment
+ * @returns {Boolean}
+ * checks if amendment has coverEndDate or facility value amendment
+ * if both or just 1, checks if both or 1 have been declined and returns true
+ * else returns false
+ */
+const ukefDecisionRejected = (amendment) => {
+  const { DECLINED } = CONSTANTS.DECISIONS.UNDERWRITER_MANAGER_DECISIONS;
+  // checks for boolean variable for which values amendment is changing
+  const coverEndDateDecision = amendment.ukefDecision.coverEndDate;
+  const facilityValueDecision = amendment.ukefDecision.value;
+
+  // if both then checks both are declined
+  if (amendment.changeCoverEndDate && amendment.changeFacilityValue) {
+    if (coverEndDateDecision === DECLINED && facilityValueDecision === DECLINED) {
+      return true;
+    }
+  } else if (coverEndDateDecision === DECLINED || facilityValueDecision === DECLINED) {
+    // else if only 1, checks either is declined
+    return true;
+  }
+  return false;
+};
+
+/**
+ * @param {Object} amendment
+ * @param {String} decisionType
+ * @returns {Boolean}
+ * checks if amendment has declined or approved with conditions and returns true if so
+ */
+const underwritingDecisionCheck = (amendment, decisionType) => {
+  const { ukefDecision } = amendment;
```

Comment:
you can pass directly the `ukefDecision` to this function, rather than the entire `amendment` object

Reply:
done